exporter/awss3exporter/README.md

```diff
@@ -42,6 +42,8 @@ Marshaler determines the format of data sent to AWS S3. Currently, the following
 - `otlp_json` (default): the [OpenTelemetry Protocol format](https://github.com/open-telemetry/opentelemetry-proto), represented as json.
 - `sumo_ic`: the [Sumo Logic Installed Collector Archive format](https://help.sumologic.com/docs/manage/data-archiving/archive/).
   **This format is supported only for logs.**
+- `body_log`: In case you only want to wrap logs body for simple to use Athena query.
```

Comment:
Is Athena the only use case?

Reply:
No, it's not the only case. There are numerous use cases where focusing solely on the body content can be beneficial:
- This approach helps minimize the transformation cost from JSON to aggregated relational data, facilitating the creation of dashboards. 
- when combined with a body-filtering processor, it becomes conducive to archiving data for long-term use. 
- By using logs as measure data, this strategy also contributes to a reduction in the main system database write I/O.

BTW: I updated the document to make it more general.